examples/js/loaders/BasisTextureLoader.js

```diff
@@ -115,25 +84,58 @@ THREE.BasisTextureLoader.prototype = Object.assign( Object.create( THREE.Loader.
 
 			}
 
-			this._createTexture( buffer, url )
-				.then( onLoad )
+			this._createTexture( [ buffer ] )
+				.then( function ( _texture ) {
+
+					texture.copy( _texture );
+					texture.needsUpdate = true;
+
+					if ( onLoad ) onLoad( texture );
+
+				} )
 				.catch( onError );
 
 		}, onProgress, onError );
 
+		return texture;
+
+	},
+
+	/** Low-level transcoding API, exposed for use by THREE.KTX2Loader. */
+	parseInternalAsync: function ( options ) {
+
+		var { levels, hasAlpha, basisFormat } = options;
```

Comment:
lgtm.com revealed two minor code issues: `hasAlpha` and `basisFormat` are never used in `parseInternalAsync()`.